OutaTime/OutaTime/View Controllers/TimeCircuitsViewController.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  TimeCircuitsViewController.swift
+//  OutaTime
+//
+//  Created by Richard Gibbs on 5/20/20.
+//  Copyright © 2020 Rich Gibbs. All rights reserved.
+//
+
+import UIKit
+
+class TimeCircuitsViewController: UIViewController {
+
+
+    var currentSpeed = 0
+    var dateFormatter: DateFormatter {
+        let dateFormatter = DateFormatter()
+        dateFormatter.dateFormat = "MMM d, yyyy"
+        dateFormatter.timeZone = TimeZone(secondsFromGMT: -25200)
+        return dateFormatter
+    }
+    
+
+    
```

Comment:
Let's work on not having a ton of whitespace between our code.
Have 1-2 lines of white space is fine as it allows for readability. But having too much makes files unnecessarily longer.